src/pages/commits/ActiveCommits/BuildVariantCard.tsx

```diff
@@ -50,13 +62,25 @@ export const BuildVariantCard: React.FC<Props> = ({
   }
   return (
     <Container>
-      <Label key={buildVariantDisplayName}>{buildVariantDisplayName}</Label>
+      <Label key={buildVariantId}>{buildVariantDisplayName}</Label>
```

Comment:
This also isn't unique since multiple versions will have the same the buildVariant.  
```suggestion
      <Label key={`${buildVariant}_${versionId}`}>{buildVariantDisplayName}</Label>
```

Reply:
We could actually omit this key since it's not being included in an array